iOS/FlowWalletKit/Sources/Network.swift

```diff
@@ -59,7 +59,7 @@ extension KeyIndexerResponse {
 
 public enum Network {
     public static func findAccount(publicKey: String, chainID: Flow.ChainID) async throws -> KeyIndexerResponse {
-        guard let host = chainID.keyIndexer?.absoluteString, let url = URL(string: host + publicKey) else {
+        guard let host = chainID.keyIndexer?.absoluteString, let url = URL(string: host + "key/" + publicKey) else {
```

Comment:
Could we update this endpoint instead ?
https://github.com/Outblock/Flow-Wallet-Kit/blob/f26e8012badfc874c2fa2264b45bdb50e55c4f19/iOS/FlowWalletKit/Sources/Extension.swift#L54-L62

Reply:
I don't understand what you mean？ @lmcmz